src/shared/helpers/formatters/numberFormatter.ts

```diff
@@ -18,8 +18,12 @@
     Formats the input number to comma separated representation
     eg: 10000 -> 10,000
 */
-export const getCommaSeparatedNumber = (input: number) => {
-  return input.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+export const getCommaSeparatedNumber = (
```

Comment:
I guess a question for me is whether to keep calling this function `getCommaSeparatedNumber` or change its name to `getFormattedNumber` or `formatNumber`.

Reply:
But how many types of formatted numbers will we have?

Reply:
🤷‍♂️